cli/config/credentials/env_store_test.go

```diff
@@ -0,0 +1,74 @@
+package credentials
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/docker/cli/cli/config/types"
+	"gotest.tools/v3/assert"
+)
+
+func TestEnvStore(t *testing.T) {
+	envConfig := map[string]types.AuthConfig{
+		"https://example.com": {
+			Email:         "something-something",
```

Comment:
We should probably update tests to use `UserName` - the `Email` field has been deprecated for a long time (but is still there 🤔)